Deprecated Security and Compliance – Content Search V2 integration (#44387)
demisto/content: Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2_description.md

# O365 - Security And Compliance - Content Search V2

**⚠️ Deprecation Notice**

> - This integration is **deprecated**. Customers should transition to the **Microsoft Graph Security** integration for the supported replacement functionality. For a detailed list of command replacements, refer to the **[integration documentation](https://xsoar.pan.dev/docs/reference/integrations/security-and-compliance-v2)**.

Reviewer: ```suggestion
- This integration is deprecated. Use the Microsoft Graph Security integration for supported replacement functionality. For a list of command replacements, see the [integration documentation](https://xsoar.pan.dev/docs/reference/integrations/security-and-compliance-v2).
```
